src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -606,15 +607,32 @@ class ContentSidebar extends PureComponent<Props, State> {
         }
     }
 
+    /**
+     * Adds a comment to the state and return the new comment.
+     *
+     * @param {Function} resolve - Resolver function for onCommentCreate() returned Promise
```

Comment:
I would rename this so that we know it's the callback from the activity feed.

Reply:
This is not the callback from the AF. I'll clean this up a tad to that instead of taking a resolver function, it just adds to the comment entries.